### PR TITLE
feat: support Google Domain ddns

### PR DESCRIPTION
**Pull request by yin1999**
Fixes: #283.

## config

image

## log

image

### CODE REVIEW

File: web/writing.html [hunk 1]
```diff
@@ -417,42 +423,31 @@ <h5 class="portlet__head">Webhook</h5>
...
+
+    const dnsConfig = {
+      dnspod: dnspodCheckedFun,
+      cloudflare: cloudflareCheckedFun,
+      huaweicloud: huaweicloudCheckedFun,
+      callback: callbackCheckedFun,
+      baiducloud: baiducloudCheckedFun,
+      porkbun: porkbunCheckedFun,
+      godaddy: godaddyCheckedFun,
+      googledomain: googleDomainCheckedFun
```
Comment (yin1999) [inline web/writing.html:445]: use key-value pairs to reduce the lines

File: web/writing.html [hunk 2]
```diff
@@ -417,42 +423,31 @@ <h5 class="portlet__head">Webhook</h5>
       document.getElementById("dns_help").innerHTML = "<a target='_blank' href='https://developer.godaddy.com/keys'>创建 API KEY</a>"
     }
 
-    var dnsName = '{{$.DNS.Name}}'
+    function googleDomainCheckedFun() {
+      document.getElementById("DnsID").disabled= false
+      if (beforeDnsID)
+        document.getElementById("DnsID").value= beforeDnsID
+      document.getElementById("dnsIdLabel").innerHTML = "Username"
+      document.getElementById("dnsSecretLabel").innerHTML = "Password"
```
Comment (yin1999) [inline web/writing.html:431]: Not sure for using `用户名` or `Username` (Has been localized)

**Comment by yin1999** [comment]
A question here, we just used `jQuery` here, if that's ok to migrate to [ES 5](https://262.ecma-international.org/5.1/). If so, I'd like to have a try (after #367 is done).

**Comment by jeessy2** [comment]
> A question here, we just used `jQuery` here, if that's ok to migrate to [ES 5](https://262.ecma-international.org/5.1/). If so, I'd like to have a try (after #367 is done).

ok. We shoud consider the i18n of the web interface.

**Pull request merged**